pkg/R/SQLContext.R

```diff
@@ -36,12 +36,14 @@ jsonFile <- function(sqlCtx, path) {
 #' @export
 
 # TODO: Implement saveasParquetFile and write examples for both
-parquetFile <- function(sqlCtx, path) {
-  # Allow the user to have a more flexible definiton of the text file path
-  path <- normalizePath(path)
-  # Convert a string vector of paths to a string containing comma separated paths
-  path <- paste(path, collapse=",")
-  sdf <- callJMethod(sqlCtx, "parquetFile", path)
+parquetFile <- function(sqlCtx, ...) {
+  # Iterate through paths
+  paths <- list(...)
+  for (i in seq_along(paths)) {
+    # Allow the user to have a more flexible definiton of the text file path
+    paths[[i]] <- normalizePath(paths[[i]])
+  }
```

Comment:
I think the reason it doesn't work is that the Java method signature looks different for `parquetFile` when compared to something like `textFile`. One thing we could do is add unit tests with multiple files for all read methods so the code paths get tested better.